modules/eks/actions-runner-controller/README.md

```diff
@@ -56,16 +65,45 @@ components:
               scale_down_factor: 0.5
             resources:
               limits:
-                cpu: 50m
-                memory: 100Mi
+                cpu: 200m
+                memory: 1Gi
               requests:
-                cpu: 10m
-                memory: 30Mi
+                cpu: 100m
+                memory: 128Mi
             webhook_driven_scaling_enabled: false
             pull_driven_scaling_enabled: false
             labels:
               - "Ubuntu"
               - "core-otto"
+          app-runner:
+            type: "repository" # can be either 'organization' or 'repository'
+            dind_enabled: true # If `true`, a Docker sidecar container will be deployed
+            # To run Docker in Docker (dind), change image from summerwind/actions-runner to summerwind/actions-runner-dind
+            image: summerwind/actions-runner-dind 
+            scope: "example/app"
+            scale_down_delay_seconds: 300
+            min_replicas: 1
+            max_replicas: 2
+            busy_metrics:
+              scale_up_threshold: 0.75
+              scale_down_threshold: 0.25
+              scale_up_factor: 2
+              scale_down_factor: 0.5
```

Comment:
Why are we not using webhook driven scaling?

Reply:
adding webhook driven scaling